kms.tf

```diff
@@ -7,6 +7,14 @@ data "aws_caller_identity" "kms" {
 }
 
 data "aws_iam_policy_document" "trail_kms" {
```

Comment:
We should create an issue for proper security findings validation and follow the least privileges principle. In issue, we can mention best practices for [AWS KMS key cloudtrail configuration](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/create-kms-key-policy-for-cloudtrail.html)